build-system/compile/compile.js

```diff
@@ -272,7 +272,7 @@ function compile(
       assume_function_wrapper: true,
       // Transpile from ES6 to ES5 if not running with `--esm`
       // otherwise transpilation is done by Babel
-      language_in: argv.esm ? 'ECMASCRIPT_2017' : 'ECMASCRIPT6',
+      language_in: 'ECMASCRIPT_2018',
```

Comment:
Is the comment in lines 273-274 still accurate?

Reply:
Updated.